### ISSUE TITLE
Missing Light Themes

### ISSUE THREAD

**Issue by chriskempson** [chriskempson/base16#42]
Base16 used to force the creation of light and dark themes by automatically generating both variants. This is no longer the case. There were two problems with this approach.

- Some themes don't look good light and some don't look good dark. A scheme author might be tempted to try and make a theme that works for both variants which would limit the range of themes that can be created. 
- When colours that work well on a dark background are applied to a light background they often require tweaking (boosting saturation or brightness) to continue to look good on a light background. This was not possible previously.

Since this change requires the creation of additional scheme files to produce the light variants, all the light variants are initially missing but will be added back at the scheme authors request or by user request.

Please comment on which themes you think work well as light variants.

**Comment by thenewvu** [comment]
Hello,

I've used `google-light`, I hope it come back with below reasons:
* It's minimal
* Its colors makes me happy ( I don't know why, but the dark version doesn't)
* It's fit with my terminal color scheme (which is `google-light` too)

**Comment by chriskempson** [comment]
I took a look at the Google theme and saw its colours worked well as both a light and dark theme so have added the light theme and kept the dark theme too.

**Comment by tomauty** [comment]
`tomorrow`, `yesterday`, and `twilight` are my favorite light variants

**Comment by chriskempson** [comment]
Tomorrow has been added recently. Yesterday was just tomorrow from what can tell so was not brought across. I'll take a look at twilight soon.

**Comment by cafink** [comment]
Shapeshifter-light is my favorite Base-16 color scheme.

**Comment by sqrt** [comment]
I vote for pop and eighties, my favorites since the have the best contrast IMHO.

**Comment by wincent** [comment]
grayscale-light worked well.

Kind of sad about this change, but at least you have a rationale for it. (I had [a set-up](https://youtu.be/QcOxU1sOOuw) that enabled me to set color schemes from the terminal with [a wrapper script](https://github.com/wincent/wincent/blob/a4f1e80cb504099464b0d9850bfd76bb099ce613/roles/dotfiles/files/.zsh/colors) around base16-shell and have Vim [pick them up automatically](https://github.com/wincent/wincent/blob/a4f1e80cb504099464b0d9850bfd76bb099ce613/roles/dotfiles/files/.vim/after/plugin/color.vim). That system relied on consistent/predictable names for all the `.dark` and `.light` variants. I can rework all that to instead to drive the shell colors from inside Vim using [`g:base16_shell_path`](https://github.com/chriskempson/base16-vim/blob/e1af5dcd7caff773eba4b5f4968a6504d5f16375/colors/base16-tomorrow-night.vim#L9-L13), but it loses some of the niceties of the wrapper script, and feels a bit backwards because Vim should be subordinate to the Terminal and not the other way around; ie. if I set a scheme in the shell it should not get overridden just because I edit a file).

**Comment by expipiplus1** [comment]
For people just bumping into this, 8698140 is the commit to wait at until the light schemes come back.

**Comment by raw1z** [comment]
harmonic16 is also great in its light version

**Comment by iphilgood** [comment]
I vote for eighties and ocean 🌊🌊🌊

**Comment by chriskempson** [comment]
![image](https://cloud.githubusercontent.com/assets/473831/16702604/1bc0e1ce-455f-11e6-929f-a617d7fa78f0.png)
Twilight doesn't work so well as a light theme (especially the yellow) though feel free to contact David Hart (http://hart-dev.com) and ask him to create a light variant.

**Comment by chriskempson** [comment]
@cafink Seems Shapeshifter doesn't work well as a dark theme (comments are hard to read) so have made this light only. Will upload soon.

![image](https://cloud.githubusercontent.com/assets/473831/16702807/5d7b9806-4560-11e6-913c-ade3f153c754.png)

**Comment by chriskempson** [comment]
@raw1z Harmonic16 works well as both a light and dark theme so will add the light one soon.

**Comment by chriskempson** [comment]
@sqrt, @iphilgood I created pop, eighties and ocean so will create new light variants optimised for light background soon. Should look better than was possible before 😄

**Comment by iphilgood** [comment]
@chriskempson Thank you so much! :+1: I really love those themes and I work with them every day. Could not imagine how it would be without them. :wink:

**Comment by chriskempson** [comment]
@wincent Yes, the predictable naming did have some advantages but forcing each theme to be light/dark also had disadvantages. I'm a bit sad to break this too but ultimately I think its for the best. Will be adding Grayscale Light soon.

**Comment by lencioni** [comment]
@wincent I was using a similar script as you to keep my Vim colors in sync with my terminal colors. After updating my Base16 to include this change, I adjusted my script to extract the background color from the theme being switched to and compute its luma to determine its perceptual lightness. This allows me to infer "dark" or "light". Perhaps you might find this useful?

https://github.com/lencioni/dotfiles/commit/f8a65296e72183bcd6c5b845a0556e6110019c77

**Comment by raw1z** [comment]
@chriskempson  Thank you very much

**Comment by rshurts** [comment]
**apathy** looks great in light. Can this be added?

Also with a breaking change like this can we get tagged git versions? That way I can revert to an older version when things break.

**Comment by exchgr** [comment]
`bright` was one of my favorite light themes, especially iterm and vim.

**Comment by chriskrycho** [comment]
@chriskempson, [a few months ago](https://github.com/chriskempson/base16/issues/42#issuecomment-229461411):

> Tomorrow has been added recently. Yesterday was just tomorrow from what can tell so was not brought across. I'll take a look at twilight soon.

FWIW, `yesterday` may have been `tomorrow` but `yesterday-night` and `yesterday-bright` are distinctly *not* like `tomorrow`. If you could bring those across, or at least list them in the unclaimed, that would be great. 😄

**Comment by jgallen23** [comment]
any chance we can get ocean light?  Its listed here: https://chriskempson.github.io/base16/, but can't find it anywhere

**Comment by Cutuchiqueno** [comment]
In my case I really would like to have monokai-light

**Comment by JemBijoux** [comment]
If I were going to vote, I'd mention that `flat` has always been my favourite. flat-dark is my default, but I'd be stoked to see flat-light make a comeback also pls.
